app/tooling/sales-admin/commands/vault_init.go

```diff
@@ -0,0 +1,96 @@
+package commands
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"log"
+	"os"
+	"strings"
+
+	"github.com/ardanlabs/service/foundation/vault"
+)
+
+const credentialsFileName = "/tmp/vault-init.json"
+
+// VaultInit sets up a newly provisioned vault instance
+func VaultInit(vaultConfig vault.Config) error {
+	var (
+		initResponse = &vault.InitResponse{}
+		ctx          = context.Background()
+	)
+
+	vaultSrv, err := vault.New(vault.Config{
+		Address:   vaultConfig.Address,
+		Token:     vaultConfig.Token,
+		MountPath: vaultConfig.MountPath,
+	})
+	if err != nil {
+		return fmt.Errorf("constructing vault: %w", err)
+	}
+
+	if _, err := os.Stat(credentialsFileName); err == nil {
+		log.Printf("credentials file exists, loading from %s", credentialsFileName)
+
+		initResponseJson, err := os.ReadFile(credentialsFileName)
+		if err != nil {
+			log.Fatalf("Error reading %s file: %s", credentialsFileName, err)
+		}
+
+		err = json.Unmarshal(initResponseJson, &initResponse)
+		if err != nil {
+			log.Fatalf("Error Unmarshalling json: %s", err)
+		}
+	} else if errors.Is(err, os.ErrNotExist) {
```

Comment:
Don't use an else clause unless you have ABSOLUTELY no choice. A switch is a better option.